backend/app/scrape/routes.py

```
@@ -1,15 +1,18 @@
 from flask import jsonify, request, make_response
 from app.scrape import bp
 from app.utils import verify_auth_header
+from app.extensions import scraper
 from playwright.sync_api import sync_playwright
 from bs4 import BeautifulSoup
 from app.config import Config
+import pprint
```

Ruff (0.11.9)

`pprint` imported but unused

Remove unused import: `pprint`

(F401)